rdr_service/genomic/genomic_job_components.py

```diff
@@ -2543,6 +2638,32 @@ def _get_new_c1_participants(self, from_date):
 
         return list([list(r) for r in zip(*result)])
 
+    def _get_long_read_participants(self):
+        """
+        Retrieves participants based on filters that have
+        been denoted to use in the long read pilot program
+        """
+        with self.member_dao.session() as session:
+            result = session.query(GenomicSetMember).join(
+                ParticipantSummary,
+                GenomicSetMember.participantId == ParticipantSummary.participantId,
+            ).join(
+                ParticipantRaceAnswers,
+                ParticipantRaceAnswers.participantId == ParticipantSummary.participantId,
+            ).join(
+                Code,
+                ParticipantRaceAnswers.codeId == Code.codeId,
+            ).filter(
+                Code.value == 'WhatRaceEthnicity_Black',
+                GenomicSetMember.genomeType == 'aou_wgs',
+                GenomicSetMember.genomeType != 'long_read',
```

Comment:
Is the intention with this to filter out any participants that were already submitted for a long-read manifest?

Reply:
Correct. Since we have to run this multiple times, like once a month, don't want to pick up the ones that have been denoted with that genome type already.

Reply:
I think there would need to be something more for filtering out `aou_wgs` genome type set members that already have a corresponding `long_read` genome type set member. This will pick up a set member with a genome type of `aou_wgs` the first time around. And after that first pass (if I'm understanding this right), each participant used would have two set members: one for `aou_wgs` and the other for `long_read`. Then the second time this query could still use the `aou_wgs` types that it used before because their type isn't `long_read`.